src/jm_view_server/app.py

```
@@ -0,0 +1,976 @@
+                op = self.jm_option
+
+            op.download_album(album_id)
+        except Exception as e:
+            self.jm_log_msg_queue.put(f'下载失败: {e}')
+        finally:
+            self.jm_log_msg_queue.put(end)
+
+    def open_directory(self, directory):
+        import subprocess
+        import sys
+        # Flask 的 <path:...> 转换器会吃掉前导 '/'，导致 mac/linux 绝对路径
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)